### PR TITLE
POC/k8s: Use an unstructured dashboard watcher rather than thema

### PR DESCRIPTION
**Pull request by ryantxu**
In order to support all dashboards (before and after 37) we can pass all dashbaords to storage without first parsing with thema.  This will let entity_api decide if it should try validating with thema or not

### CODE REVIEW

File: pkg/codegen/jenny_crd_watcher.go [hunk 1]
```diff
@@ -31,12 +31,16 @@ func (j crdWatcherJenny) Generate(kind kindsys.Kind) (*codejen.File, error) {
 		return nil, nil
 	}
 
+	name := kind.Props().Common().MachineName
+	if name == "dashboard" {
```
Comment (ryantxu) [inline pkg/codegen/jenny_crd_watcher.go:35]: HACK to avoid typed watcher

Comment (DanCech) [reply pkg/codegen/jenny_crd_watcher.go:35]: looks like there's also a typo on line 29, should be kindsys.Custom no?

Comment (toddtreece) [reply pkg/codegen/jenny_crd_watcher.go:35]: good catch @DanCech

Comment (ryantxu) [reply pkg/codegen/jenny_crd_watcher.go:35]: updated

**Comment by sdboyer** [comment]
oooh yeah, this is an exception case we never robustly dealt with.

this hack seems fine to me for now. We'll clearly need some version of this hack long-term as part of the larger handoff of responsibility from frontend dashboard migrator to thema.

**Review by toddtreece — APPROVED** [review]

**Pull request merged**